apolloschurchapp/fastlane/Fastfile

```diff
@@ -0,0 +1,43 @@
+# This file contains the fastlane.tools configuration
+# You can find the documentation at https://docs.fastlane.tools
+#
+# For a list of all available actions, check out
+#
+#     https://docs.fastlane.tools/actions
+#
+# For a list of all available plugins, check out
+#
+#     https://docs.fastlane.tools/plugins/available-plugins
+#
+
+# Uncomment the line if you want fastlane to automatically update itself
+update_fastlane
+
+before_all do
+  ensure_git_branch
+  ensure_git_status_clean
+  git_pull
+end
+
+platform :ios do
+  desc 'Fetch certificates and provisioning profiles'
+  lane :certificates do
+    match(app_identifier: 'com.differential.apollos', type: 'development', readonly: true)
+    match(app_identifier: 'com.differential.apollos', type: 'appstore', readonly: true)
+  end
+
+  desc "Push a new beta build to TestFlight"
+  lane :beta do
+    increment_build_number(xcodeproj: "apolloschurchapp.xcodeproj")
```

Comment:
we might need to rethink this bit. If we increment the build number, but don't commit and push, we'll end up incrementing to the same value every time. We might need a way to look at the last published build number, or use the number of commits in the repo, or something like that.

Reply:
for newspring we did it by date. I can have fastlane actually do a commit and push. for now I just wanna test the manual deploy.